lang/src/org/partiql/lang/syntax/LexerConstants.kt

```diff
@@ -535,12 +550,18 @@ enum class OperatorPrecedenceGroups(val precedence: Int) {
  */
 @JvmField internal val OPERATOR_PRECEDENCE = mapOf(
     // set operator group
+    "union" to OperatorPrecedenceGroups.SET.precedence,
```

Comment:
I think this map is missing the `*_distinct` variants:
- union_distinct
- intersect_distinct
- except_distinct
- outer_union_distinct
- outer_intersect_distinct
- outer_except_distinct